src/lightning/pytorch/strategies/fsdp.py

```diff
@@ -200,12 +200,6 @@ def root_device(self) -> torch.device:
     def num_processes(self) -> int:
         return len(self.parallel_devices) if self.parallel_devices is not None else 0
 
-    @property
-    def process_group(self) -> Optional[ProcessGroup]:
```

Comment:
it is still public API

Reply:
The FSDP strategy is experimental: https://github.com/Lightning-AI/lightning/blob/master/src/lightning/pytorch/strategies/fsdp.py#L92